### PR TITLE
Internal ci 2021.1 110

### PR DESCRIPTION
**Pull request by gundaarx**
Made changes for InternalCI and DLDT build is replaced with docker.

### CODE REVIEW

File: tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino [hunk 1]
```diff
@@ -2,33 +2,44 @@ ARG OS_VERSION=18.04
...
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-igc-core_1.0.2-1787_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-igc-opencl_1.0.2-1787_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-opencl_19.15.12831_amd64.deb && \
-    wget https://github.com/intel/compute-runtime/releases/download/19.15.12831/intel-ocloc_19.15.12831_amd64.deb && \
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN wget https://apt.repos.intel.com/openvino/2021/GPG-PUB-KEY-INTEL-OPENVINO-2021 && \
+    apt-key add GPG-PUB-KEY-INTEL-OPENVINO-2021 && rm GPG-PUB-KEY-INTEL-OPENVINO-2021 && \
+    cd /etc/apt/sources.list.d && \
+    echo "deb https://apt.repos.intel.com/openvino/2021 all main">intel-openvino-2021.list && \
```
Comment (smkarlap) [inline tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino:26]: Wasn't '[trusted=yes]' needed in this line?

Comment (gundaarx) [reply tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino:26]: That is required only if we have proxy.

**Comment by smkarlap** [comment]
LGTM

File: tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino [hunk 2]
```diff
@@ -2,33 +2,44 @@ ARG OS_VERSION=18.04
...
+ARG OPENVINO_VERSION=2021.1
 
 ADD scripts /tmp/scripts
 RUN /tmp/scripts/install_ubuntu.sh -p $PYTHON_VERSION -d EdgeDevice && \
     /tmp/scripts/install_deps.sh -p $PYTHON_VERSION -d EdgeDevice
 
 RUN apt update && apt install -y libnuma1 ocl-icd-libopencl1 && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN /tmp/scripts/install_openvino.sh -o ${OPENVINO_VERSION} && \
```
Comment (suryasidd) [inline tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino:14]: If we are not using install_openvino.sh, can we remove it or are we still using it for something else

Comment (suryasidd) [inline tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino:14]: If we are not using install_openvino.sh, can we remove it or are we still using it for something else

**Pull request closed without merging**